### ISSUE TITLE
statement of need (readme)

### ISSUE THREAD

**Issue by corybrunson** [fauvernierma/survPen#15]
The package documentation (`?survPen`) and JOSS paper provide clear context and motivation for the package. To help users who first encounter the package on GitHub or elsewhere (and to satisfy JOSS guidelines) could you summarize these into a short statement of need in the `README`? Such a summary would also be useful to include in the `Description` field of the `DESCRIPTION` file, before the technical summary already there, for people who find the package on CRAN.

_Part of [this JOSS review](https://github.com/openjournals/joss-reviews/issues/1434)._

**Comment by fauvernierma** [comment]
Hi,

I think the following 00eed48
799451a solve the issue

Thanks

**Comment by corybrunson** [comment]
Thanks! The new text looks good in the README, but it should be shortened to a single paragraph for the `Description` field. The CRAN article on [Writing R Extensions](https://cran.r-project.org/doc/manuals/r-release/R-exts.html#The-DESCRIPTION-file) contains complete details, including this bit:

> The mandatory ‘`Description`’ field should give a _comprehensive_ description of what the package does. One can use several (complete) sentences, but only one paragraph. It should be intelligible to all the intended readership (e.g. for a CRAN package to all CRAN users).

**Comment by fauvernierma** [comment]
Hi,

Yes, Travis made me realize this pretty clearly by not wanting to load survPen :)

It is ok now 48ca526b80bc8400db20a24d3321a3fc912fe6a6 I think

Thanks

**Comment by corybrunson** [comment]
Yep, i think it looks good now!
